### DISCUSSION TITLE
how to regenerate responsive images for already uploaded images ?

### DISCUSSION THREAD

**Discussion by ctf0** [spatie/laravel-medialibrary#2038] [Q&A] [answered]
atm `media-library:regenerate` works only for conversions 😢

**Comment by freekmurze** [comment]
I'd accept a PR that adds support for responsive images

**Comment by ctf0** [comment]
sure np, could u tell me where to start ? or what are the main steps for responsive images to be created ? 

specially that the command already works with responsive images as long as its under a conversion (at least according to https://github.com/spatie/laravel-medialibrary/issues/1115)

**Comment by ctf0** [comment, marked as answer]
ref https://github.com/spatie/laravel-medialibrary/discussions/2038

lets continue over there
